packages/clerk-js/src/ui/components/Subscriptions/SubscriptionDetails.tsx

```diff
@@ -92,10 +139,17 @@ export function SubscriptionDetails({
             role='list'
             sx={t => ({
               display: 'grid',
-              rowGap: t.space.$4,
+              rowGap: t.space.$6,
               padding: t.space.$4,
             })}
           >
+            <Text
+              elementDescriptor={descriptors.subscriptionDetailCaption}
+              variant={'caption'}
+              localizationKey={localizationKeys('__experimental_commerce.availableFeatures')}
+              colorScheme='secondary'
+            />
```

Comment:
Can this move outside of the list. This is leading to invalid markup with the `p` tag, being a direct descendant of a `ul`.

![Screenshot 2025-04-28 at 2 55 22 PM](https://github.com/user-attachments/assets/a5d3fe54-92be-49fd-8902-1ea9a5db6a0d)